src/prisma/migrations/20230921001241_/migration.sql

```diff
@@ -0,0 +1,339 @@
+-- CreateTable
+CREATE TABLE `account_bans` (
```

Comment:
I dont think server tables should be a migration of AAC, we should be migrating only what we need for the aac to work, which does not include server tables